### ISSUE TITLE
Rotation in epaper-breakout-top-jlcpcb.csv maybe not correct

### ISSUE THREAD

**Issue by Li-Tianming** [vroland/epdiy#96]
In epaper-breakout-top-jlcpcb.csv, [U5 component](https://github.com/vroland/epdiy/blame/master/hardware/epaper-breakout/gerbers/epaper-breakout-top-jlcpcb.csv#L74) rotation is 90 degree, 

but in JLC's component description, it's  original orientation is like below, 

[JLC's STM search system](https://www.jlc.com/portal/smtComponentList.html)

![QFN-38_EP_18 0X31 4X1 27P](https://user-images.githubusercontent.com/87600827/126993760-83595511-b6b3-4960-ac36-c9b319d75b77.Png)

![img_jlc_footprint_orientation](https://user-images.githubusercontent.com/87600827/126993766-fa68207e-d684-4d60-a4d7-a92faffda5d2.png)

If i use 90 as rotation, in JLC_assist PC program preview, it will looks like this, obvious it's not correct.
![img_jlc_assist_preview](https://user-images.githubusercontent.com/87600827/126993769-3a0a8127-1e99-43ec-a186-7c605ae31194.png)

so i think the correct rotation maybe is 270 degree

**Comment by vroland** [comment]
Hi, so far I always soldered the ESP32 module by hand, so this is probably just a mistake. Unfortunately, rotations as exported by KiCad do not correspond to JLCPCB for some packages. I tried to fix most of them, but the ESP is not a standard package.
If 270° looks better, it's probably correct :) Fortunately, they usually fix such things when it's as obvious as with the ESP.
If you want, feel free to submit a pull request with the fixed rotation.

**Comment by Li-Tianming** [comment]
thanks for the reply.  

i am trying to make thing correct and works, if thing goes well, i will create a pull request to update the jlc's SMT file.

**Comment by 2dom** [comment]
Yeah ... same here.  JLCPCB shows all the parts (even resistors etc) shifted with v5. Not usable for ordering SMT assembly unfortunately :(

**Comment by vroland** [comment]
Hm, resistors, capacitors and diodes always where correct for me... That's strange. Is that still the case when you manually export the position file again from KiCad?

**Comment by Li-Tianming** [comment]
![img11](https://user-images.githubusercontent.com/87600827/132119467-15882386-c87c-44d4-bbe4-439252b9b8d8.png)


everything works pretty fine

**Comment by mcer12** [comment]
@Li-Tianming please note that the JLCPCB assembly is kinda broken so it sometimes shows the footprints wrong. They always manually fix it and send you a screenshot so that you can check. Don't worry about it :)
